plugins/modules/vmware_guest_tools_upgrade.py

```diff
@@ -76,13 +76,24 @@
 '''
 
 EXAMPLES = '''
+- name: Get VM UUID
+  vmware_guest_facts:
+    hostname: "{{ vcenter_hostname }}"
+    username: "{{ vcenter_username }}"
+    password: "{{ vcenter_password }}"
+    datacenter: "{{ datacenter }}"
+    folder: "/{{datacenter}}/vm"
```

Comment:
```suggestion
    folder: "/{{ datacenter }}/vm"
```

Reply:
thank you! Added another PR for improving the source: https://github.com/ansible-collections/vmware/pull/22